api/hypershift/v1beta1/nodepool_types.go

```diff
@@ -931,6 +931,11 @@ type OpenStackNodePoolPlatform struct {
 	//
 	// +optional
 	ImageName string `json:"imageName,omitempty"`
+
+	// AvailabilityZone is the failure domain identifier where the VM should be attached to.
```

Comment:
Do we have any restrictions we can apply here? A valid character set or pattern? A minimum or maximum length?

Reply:
I'll copy paste the doc & validation from CPMS API :smiley: